LibreNMS/Alert/Transport/Pagerduty.php

```diff
@@ -42,9 +42,8 @@ public function deliverAlert($obj, $opts)
         } else {
             $protocol['event_type'] = 'trigger';
         }
-        foreach ($obj['faults'] as $fault => $data) {
-            $protocol['details'][] = $data['string'];
-        }
+        $protocol['details'] = end($obj['faults']);
```

Comment:
Won't this just store the last fault when it can have multiple?

Reply:
Whoops, you are right, I didn't realize that it would send out these alarms bundled by device/rule.

Well there's an easy fix for that with the current code, I'll let someone else write and pull it if they like.  It doesn't parse very nicely on the mobile app though, and thus gets very wall-of-texty with any more than a couple faults.

I'm just going to kill this PR for the time being and do a cleaner rewrite, possibly with PD events api v2.  May make a new PR later when that's ironed out.